apps/studio/src/components/AppNavbar.tsx

```diff
@@ -33,22 +33,24 @@ export function AppNavbar(): JSX.Element {
         transition="padding 0.1s"
       >
         <Flex alignItems="center">
-          <IconButton
-            mr="0.5rem"
-            as={NextLink}
-            href={DASHBOARD}
-            variant="clear"
-            aria-label="Back to dashboard"
-            icon={
-              <Image
-                src="/assets/isomer-logo-color.svg"
-                height={24}
-                width={22}
-                alt="Back to dashboard"
-                priority
-              />
-            }
-          />
+          <Tooltip label={"Back to sites"} placement="right">
```

Comment:
Will this also show on the "All sites" page? That might be confusing

![Screenshot 2024-12-20 at 9.39.22 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/AWVfQY5lNPJPEvYNU3KB/4ce39d0c-fd71-4f40-ad75-6c7c1c69b9dc.png)

Reply:
It does. Currently there's already an existing link to do that without the tooltip, so IMO not adding can be more confusing because users click and "nothing happens". Perhaps a better improvement is to remove the link altogether?

Reply:
updated for homepage to not be a button and not be clickable